js/stdlib/ts/ALGO.ts

```diff
@@ -159,7 +159,7 @@ export type Backend = IBackend<AnyALGO_Ty> & {_Connectors: {ALGO: {
 type BackendViewsInfo = IBackendViewsInfo<AnyALGO_Ty>;
 type BackendViewInfo = IBackendViewInfo<AnyALGO_Ty>;
 
-export type ContractInfo = BigNumber;
```

Comment:
We actually do want this and need to find the fix another way. If it stays `number`, then things will break once we get an application where JS numbers are imprecise

Reply:
I agree, but without this it will break existing functionality; getInfo() will start returning a BigNumber on Algorand instead of a number and so many frontends that upgrade to this version will most likely run into difficulties because of that. For context, I discussed this with Dan Burton and he recommended against introducing breaking changes.

Taking all that into account, should I go back to returning a BigNumber (along with the required changes to the documentation if any) with the understanding that the release this is included in is accompanied by an announcement noting these changes to getInfo()?

Reply:
The documentation has always said that it returns a UInt and that a UInt is either a BigNumber or a number, thus this change is not backwards incompatible. Furthermore, we KNOW this change is broken, because we KNOW that a number cannot hold the correct values.

Reply:
Alright, I'll get it changed back and fix up those examples that were failing